tests/dataimportcron_test.go

```diff
@@ -72,6 +74,16 @@ var _ = Describe("DataImportCron", Serial, func() {
 
 		By("[AfterEach] Restore the profile")
 		Expect(utils.UpdateStorageProfile(f.CrClient, scName, *originalProfileSpec)).Should(Succeed())
+
+		// Clean up existing dataimportcrons in the environment that we might have switched
+		l, err := labels.Parse(common.DataImportCronLabel)
+		Expect(err).ToNot(HaveOccurred())
+		opts := &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: metav1.NamespaceAll, LabelSelector: l}}
```

Comment:
I think I'm missing something here, since we use NamespaceAll and the generic `DataImportCronLabel`, couldn't this trigger the deletion of non-testing importcrons too?

Reply:
This is exactly what we're trying to achieve - these tests will flip existing non-testing-importcrons to snapshots